config/ols-load-generator.yaml

```diff
@@ -40,18 +40,34 @@ spec:
         env:
           - name: OLS_TEST_HOST
             value: <ols-host-url>
-          - name: OLS_TEST_RUNID
-            value: <ols-test-runid>
+          - name: OLS_TEST_UUID
```

Comment:
@vishnuchalla here too the image has reference to personal repo. ( not part of this PR )

Reply:
Yes, will take them in a separate PR. Once we have things setup in prow. Open PR: https://github.com/openshift/release/pull/53601